src/zepben/evolve/streaming/grpc/connect.py

```diff
@@ -96,7 +116,8 @@ def connect(host: str = "localhost",
             client_secret: str = None,
             pkey=None,
             cert=None,
-            ca=None):
+            ca=None,
+            authenticator=None):
```

Comment:
can you just add authenticator to the docstring below, as well as on `connect_async`?

also note that authenticator will be preferred if it's provided over client creds